GOVERNANCE.md

```diff
@@ -129,7 +129,16 @@ When an agenda item has appeared to reach a consensus the moderator
 will ask "Does anyone object?" as a final call for dissent from the
 consensus.
 
+Agenda items should be tagged at least 24 hours in advance. If an item
+is added less than 24 hours before the meeting it cannot be brought to
+a vote.
```

Comment:
Maybe change to specify that it is voting only, not bringing it up for discussion.